test/lib/generators/devise_token_auth/install_generator_test.rb

```diff
@@ -47,7 +47,7 @@ class InstallGeneratorTest < Rails::Generators::TestCase
 
         test 'add primary key type with rails 5 when specified in rails generator' do
           run_generator %w[--primary_key_type=uuid --force]
-          if Rails::VERSION::MAJOR >= 5
+          if Rails::VERSION::MAJOR == 5
```

Comment:
why this?

Reply:
Ok, I've changed the generator code to allow usage of "primary_key_type" not only for Rails 5, but for everything above 5. Is that ok?